pkgs/tools/security/sequoia-tool/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchFromGitLab, rustPlatform, pkgconfig, capnproto, clang, libclang, nettle, openssl, sqlite }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "sequoia-tool";
+  version = "0.8.0";
+
+  src = fetchFromGitLab {
+    owner = "sequoia-pgp";
+    repo = "sequoia";
+    rev = "v${version}";
+    sha256 = "19k3vsaz0iqm0qxqadlfsnrnprn4lrb8078mm2if3gq7klb5csjb";
+  };
+
+  nativeBuildInputs = [ pkgconfig clang libclang ];
+  buildInputs = [ capnproto nettle openssl sqlite ];
+
+  LIBCLANG_PATH = libclang + "/lib";
+
+  cargoBuildFlags = [ "--package=sequoia-tool" ];
+
+  cargoSha256 = "03fn0vmaa8axc4yf761ds46p1hxkimbr4dzlbp018rmhgilb96p7";
+
+  meta = with stdenv.lib; {
+    description = "";
```

Comment:
"A user-friendly PGP tool" maybe?

Reply:
Yes! Must have slipped my mind, sorry about that ^^'

Reply:
Used a mix of the `Cargo.toml` description and the CLI "about" description